examples/qfn.example.tsx

```diff
@@ -1,12 +1,24 @@
 import { JsCadFixture } from "jscad-fiber"
 import { QFN } from "../lib/qfn"
 import { ExtrudedPads } from "../lib/ExtrudedPads"
+import { fp } from "@tscircuit/footprinter"
 
 export default () => {
+  const footprint = "qfn24_w8_h8_thermalpad3.2mmx3.2mm"
+  const fpJson = fp.string(footprint).json()
+  console.log(fpJson)
   return (
     <JsCadFixture zAxisUp showGrid>
-      <QFN fullWidth={4} height={0.8} thermalPadSize={2} />
-      <ExtrudedPads footprint="qfn16" />
+      <QFN
+        num_pins={24}
+        bodyWidth={8}
+        bodyLength={8}
+        thermalPadSize={{
+          width: 3.2,
+          length: 3.2,
+        }}
+      />
+      <ExtrudedPads footprint={footprint} />
     </JsCadFixture>
   )
```

Comment:
Can you add a test that uses FootprinterModel

Reply:
Or sorry, Footprinter3d

Reply:
@seveibar 
already added
examples/footprinter3d/fp-qfn56.example.tsx